blueprint/index.md

```diff
@@ -0,0 +1,311 @@
+                            "queueId": "a623d83c-96d7-4baa-af1f-d46ad6700d8b",
+                            "segmentType": "alert",
+                            "conference": false
+                        },
+                        {
+                            "segmentStart": "2021-04-01T12:28:04.569Z",
+                            "segmentEnd": "2021-04-01T12:34:19.130Z",
+                            "queueId": "a623d83c-96d7-4baa-af1f-d46ad6700d8b",
+                            "segmentType": "interact",
+                            "conference": false
+                        },
+                        {
+                            "segmentStart": "2021-04-01T12:34:19.130Z",
+                            "segmentEnd": "2021-04-01T12:43:23.306Z",
+                            "queueId": "a623d83c-96d7-4baa-af1f-d46ad6700d8b",
+                            "disconnectType": "peer",
+                            "segmentType": "hold",
+                            "conference": false
+                        }
+                    ],
+                    "metrics": [
+                        {
+                            "name": "tAlert",
+                            "value": 11050,
+                            "emitDate": "2021-04-01T12:28:04.569Z"
+                        },
+                        {
+                            "name": "tAnswered",
+                            "value": 11179,
+                            "emitDate": "2021-04-01T12:28:04.569Z"
+                        }
+                    ],
+                    "provider": "Webchat Provider",
+                    "remote": "John Doe",
+                    "requestedRoutings": [
+                        "Standard"
+                    ],
+                    "usedRouting": "Standard",
+                    "selectedAgentId": "50fcd33c-f432-4726-abc9-979f3ca01555"
+                }
+            ]
+        }
+    ]
+}
+```
+
+## Important notes
+
+The database design used in this blueprint will not store the full schema from the [POST 
+  /api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query) response, some of the data was omitted to simplify the design. The full schema can be seen at the `200 - successful operation` section of the API documentation.  
+
+The following steps outline how to get driver script running to show this design in action. It is intended as a possible starting point for developers looking to map conversation responses or other non-flat JSON data to SQL. This logic is not limited to typescript, people looking to access the SQL queries can find them in [db.ts](https://github.com/GenesysCloudBlueprints/conversation-model-to-sql-blueprint/blob/main/src/db.ts).  
+
+## Implementation steps
+
+* [Clone the repository containing the project files](#clone-the-repository-containing-the-project-files)
```

Comment:
Consider using `[toc]` here instead of building the nav by hand. The TOC generates links for all headings _after_ it.